serverless/installing-kn.adoc

```diff
@@ -20,3 +20,9 @@ include::modules/serverless-installing-cli-linux-rpm.adoc[leveloffset=+1]
 include::modules/serverless-installing-cli-linux.adoc[leveloffset=+1]
 include::modules/serverless-installing-cli-macos.adoc[leveloffset=+1]
 include::modules/serverless-installing-cli-windows.adoc[leveloffset=+1]
+include::modules/serverless-kn-cli-plugins.adoc[leveloffset=+1]
+
+.Additional resources
+
+* For instructions on using the Kafka source `kn` plugin, see xref:../serverless/event_sources/serverless-kafka-source.adoc#serverless-kafka-source[Creating a Kafka event source using the `kn` CLI].
+* For information on {ServerlessProductName} Functions, see link:https://openshift-knative.github.io/docs/docs/functions/about-functions.html[About OpenShift Serverless Functions].
```

Comment:
Remove this bullet for now; we mention functions in the release notes as dev preview, but shouldn't include anything about it in our main docs until it's tech preview.

Reply:
I somehow glazed over the fact that it was a dev and not tech preview when adding mentions of functions :unamused: Deleted.